src/components/Profile/ProfileImageField.jsx

```diff
@@ -0,0 +1,111 @@
+// React Imports
+import React, { useContext } from 'react';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import Avatar from '@mui/material/Avatar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import HideImageIcon from '@mui/icons-material/HideImage';
+import ImageIcon from '@mui/icons-material/Image';
+import Typography from '@mui/material/Typography';
+// Contexts Imports
+import { SignedInUserContext } from '../../contexts';
+
+/**
+ * profileImageFieldProps is an object that stores the props for the ProfileInputField
+ * component
+ *
+ * @typedef profileImageFieldProps
+ * @type {object}
+ * @property {Blob} profileImg - The existing file blob being used for profile
+ * card
+ * @property {React.Dispatch<any>} setProfileImg - Set function for profileImg
+ * @property {() => void} loadProfileData - Handler function for setting local
+ * state for profile card in PASS
+ * @memberof typedefs
+ */
+
+/**
+ * ProfileImageField Component - Component that creates the editable inputs fields
+ * for the Profile page
+ *
+ * @memberof Inbox
+ * @name ProfileImageField
+ * @param {profileImageFieldProps} Props - Props used for NewMessage
+ * @returns {React.JSX.Element} React component for NewMessage
+ */
+const ProfileImageField = ({ profileImg, setProfileImg, loadProfileData }) => {
+  const { session } = useSession();
+  const { profileData, removeProfileImage, uploadProfileImage } = useContext(SignedInUserContext);
+
+  const handleProfileImage = async (event) => {
+    await uploadProfileImage(session, profileData, event.target.files[0]);
+
+    loadProfileData();
+  };
+
+  const handleRemoveProfileImg = async () => {
+    if (window.confirm("You're about to delete your profile image. Do you wish to continue?")) {
+      await removeProfileImage(session, profileData);
+
+      loadProfileData();
+      localStorage.setItem('profileImageBlob', null);
+      setProfileImg(null);
+    }
+  };
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        maxWidth: '300px',
+        alignItems: 'flex-start',
+        gap: '10px'
+      }}
+    >
+      <Typography color="black">Profile Image: </Typography>
+      {profileImg ? (
+        <>
+          <Avatar
+            src={profileImg}
```

Comment:
You don't need to manually handle loading this image. The HTML `img` element at the base of this avatar will handle it far better than we can. Just pass in the URL of the image and the browser will take care of it.

Reply:
Oh, I didn't know that. Going to switch to using `Avatar` as the default icon. Just going to set the condition for the buttons.